docker/conda/environments/requirements.txt

```diff
@@ -6,4 +6,5 @@ jupyterlab
 nvidia-pyindex
 # Duplicated in conda dev to ensure parity with libprotobuf
 protobuf==4.21.*
+tritonclient==2.26.*
```

Comment:
Isnt this already included in `docker/conda/environments/cuda11.8_dev.yml`?

Reply:
not for the release container.

Reply:
To install it in the release container, you need to add it to the run requirements of `meta.yml`. Putting it here as well going to cause it to be installed twice